Synview-frontend/src/components/MentorInviteForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { useInviteMentorMutation } from "../services/apiSlice.ts";
+import { useAppSelector } from "../hooks.ts";
+import { ProjectRoles } from "../../../common/schemas.ts";
+export default function MentorInviteForm() {
+  const [invitedMentorId, setInvitedMentorId] = useState("");
+
+  const project_id = useAppSelector(
+    (state) => state.inviteMentorModal.project_id
+  );
```

Comment:
nit: Same as previous commit, should be `projectId` in the typescript/javascript world